### PR TITLE
Recalculate hashes in agility section

### PR DESCRIPTION
**Pull request by devd**
We updated the example text and forgot to update the hashes here.

### CODE REVIEW

File: specs/subresourceintegrity/spec.markdown [hunk 1]
```diff
@@ -247,15 +247,15 @@ resource in order to provide agility in the face of future discoveries.
 For example, the "Hello, world." resource described above may be described
```
Comment (fmarier) [inline specs/subresourceintegrity/spec.markdown]: Now that the resource is no longer "Hello, world.", perhaps we should rephrase that and just refer to "the resource described in the previous section may be described..."

**Comment by devd** [comment]
@fmarier wdyt?

**Comment by fmarier** [comment]
Looks good

**Comment by fmarier** [comment]
I just rebased and merged this on master.

**Pull request closed without merging**